input/fsh/who-co-Observations.fsh

```diff
@@ -111,6 +111,8 @@ Description: "WHO CO Outcome Contacts Followed Observation"
 * code = $LNC#96551-7
 * value[x] only integer
 * valueInteger 1..1
+* note 0..*
+* note.text 1..1
```

Comment:
These definitions seem unnecessary as they are the same as the original spec
https://www.hl7.org/fhir/observation.html#resource